packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -45,7 +46,20 @@ export interface CreateChannel {
 	tag: `channel-${number}`;
 }
 
-export type StressDataObjectOperations = UploadBlob | CreateDataStore | CreateChannel;
+export interface EnterStagingMode {
+	type: "enterStagingMode";
+}
+export interface ExitStagingMode {
```

Comment:
Any particular reason to do exit+boolean rather than mirror the two separate methods commit/discard that you used in the implementation?

Reply:
its easier in the stress harness, as otherwise i need separate reduces and generators